content/docs/04-clusters/03-edge/02-native.md

```diff
@@ -15,149 +15,495 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+
+5. Build the custom ISO: A custom ISO should be built with the name specified in the settings and the user data file baked inside the ISO.
+
+```
+./build-installer.sh
+```
+
+
+6. Mount this Palette Edge Installer ISO using a [bootable USB drive](http://tbd), [pixie server](http://tbd), or any other means to the bare metal appliances' primary drive (USB or CDROM). The installer will be flashed to the edge host hard disk, and the host will be shut down. The bare-metal edge host appliance is ready to be shipped to the edge location.
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="VMware Environment" key="VMware Environment">
+
+### Staging: VMware Environment (OVA)
+
+We will create an OVA file from the base Palette Edge Installer ISO for VMware environments by injecting customized settings (user data) via a secondary drive. Following are the detailed steps to generate the Palette Edge Installer variant in  OVA format. 
+
+1. Prepare cloud-init ISO with common site properties. The steps to do this can vary from platform to platform. The instructions below describe the procedure to build the cloud-init ISO on macOS. 
+ 
+
+2. Create a YAML file called ‘user-data’  with the contents from the template in the [Site Settings section](http://localhost:9000/clusters/edge/native#staging). Customize various properties as necessary.
+
+
+3. Create an empty meta-data file:
+   
+```
+ touch meta-data
+```
+
+4. Create a cloud init iso using the following command:
+
+```
+mkisofs -output ci.iso -volid cidata -joliet -
+rock user-data meta-data
+```
+ 
+ **Note** - You may need to install prerequisites using:
+ 
+```
+brew install cdrtools
+```
+This will generate an ISO file called ci.iso in the current directory. 
+
+<br />
+
+1. Upload the ci.iso file generated in the previous step to a datastore in vSphere using the vCenter console. 
+
+
+2. Download the default Palette Edge Installer image (ISO) from the appliances page on the Palette Management Console.
+
+ 
+3. Upload this ISO  to a datastore in vSphere using the vCenter console. 
+
+
+4. Create a new VM using the vCenter console. Add 2 CD drives to this VM and select the Palette Edge installer for one of them and the ci.iso 4. file for the other. 
+
+
+5. Select other settings, such as Network, Datastore, Folder, etc., as appropriate for your environment. 
+
+
+6. Power on the VM.
+
+
+7. Monitor the VM console for log messages. The installer and user data will be copied to the hard disk, and the VM will show a message “...”. 
+
+
+8. Power off the VM
+
+
+9. Edit VM settings and delete the two CD drives previously attached.
+
+
+10. Export VM as an OVF template.
+
+
+The OVA file is ready to ship to various edge locations for installation. 
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="Amazon Web Service" key="Amazon Web Service">
+
+### Staging: Amazon Web Services (AMI)
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="KVM" key="KVM">
+
+### Staging: KVM  (QCOW2)
+
+
+ To be added
```

Comment:
What is "To be added" under QCOW2?

Reply:
@jgautam will be updating in the latest.